include/SDL3/SDL_gpu.h

```diff
@@ -1651,34 +1679,30 @@ extern SDL_DECLSPEC void SDLCALL SDL_GpuUnmapTransferBuffer(
  *
  * \param device a GPU context
  * \param data a pointer to data to copy into the transfer buffer
- * \param transferBuffer a transfer buffer
- * \param copyParams a struct containing parameters specifying copy offsets and size
+ * \param transferBuffer a transfer buffer with offset and size
  * \param cycle if SDL_TRUE, cycles the transfer buffer if it is bound, otherwise overwrites the data.
  *
  * \since This function is available since SDL 3.x.x
  */
 extern SDL_DECLSPEC void SDLCALL SDL_GpuSetTransferData(
     SDL_GpuDevice *device,
     const void *data,
-    SDL_GpuTransferBuffer *transferBuffer,
-    SDL_GpuBufferCopy *copyParams,
+    SDL_GpuTransferBufferRegion *transferBuffer,
```

Comment:
Maybe rename these parameters to `transferBufferRegion` since you're passing more than just the buffer handle itself? Or better yet, just `source` and `destination`

Reply:
Went with source/destination!